### ISSUE TITLE
COAM\Penalty - View penalty payment status

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#91]
#### Description:

As an operator, once a penalty payment notification is submitted, the Review Penalty Summary page will show the current outstanding balance. Once payment received, the outstanding balance is reduced and the payment details are posted.
#### Acceptance Criteria:

As an operator, given that a payment notification has been submitted, when I view the 'Pay penalty by EFT' page, I see the outstanding balance in tCO2e and the current obligation in dollars.

- [ ] Text saying: Once your payment(s) is received and applied, the Outstanding Penalty will be updated below.
- [ ] Outstanding compliance obligation header
Under the header:
- Current obligation balance: in dollars

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?page-id=14307%3A1663&node-id=23398-455&node-type=frame&viewport=1759%2C-4811%2C0.18&t=CZ8iNtq9kg6AOlwP-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=20107%3A1497)

As an operator, given that the obligation has been fully paid, when I view the 'Pay penalty by EFT' page, I see the new, reduced balance in dollars and the payment(s) made.

- [ ] Text saying: Once your payment(s) is received and applied, the Outstanding Compliance Obligation will be updated below.
- [ ] Outstanding compliance obligation header
Under the header:
- Outstanding amount: display in dollars

Payment details:

- [ ] A header with the payment number
- [ ] Payment received date: date payment received
- [ ] Payment amount received: amount paid in dollars

As an operator, given that a payment has been received, when I view the 'Pay compliance obligation by EFT' page, I see the new, reduced balance in tCO2e and the new, reduced obligation in dollars and the payment(s) made.

- [ ] Text saying: Your payment(s) has been received and applied successfully. Your Compliance Obligation has been met.
- [ ] Outstanding compliance obligation header
Under the header:
- [ ]Display Outstanding balance: 0 tCO2e
- Current obligation balance: $0.00

Payment details:

- [ ] A header with the payment number
- [ ] Payment received date: date payment received
- [ ] Payment amount received: amount paid in dollars

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?page-id=14307%3A1663&node-id=23398-643&node-type=frame&viewport=1759%2C-4811%2C0.18&t=CZ8iNtq9kg6AOlwP-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=20107%3A1497)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #90 

#### Notes:
Same requirements as #26, the wording is different here for penalties.
-

**Comment by patrickisaac** [comment]
Refinement Notes:
- @vesselak to clean up the ticket language

**Comment by vesselak** [comment]
@nanyangpro I moved the ticket down a sprint to allow for proper clean up. Would you please take the first crack at cleaning it up? I think this ticket is about the payment instructions page and download and the end of penalty workflow to view information when the penalty has been paid as the rest is handled with #89 and #73. Thank you.

**Comment by nanyangpro** [comment]
Hi @vesselak (cc: @patrickisaac) - I've drafted the description and AC again and updated the design links. Feel free to refine further as you see fit.

The Dev Checklist requires dev's attention and effort still, thus I kept the refinement label. :)

**Comment by fviduya** [comment]
I have tested this in the DEV environment alongside #71. [Here is the test document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/EetXSoaaDkxMsa6Ze5IlQV8BThuS0hiLlon4-O0D1lhqmA?e=ynmeaN&nav=eyJoIjoiNDUxNTUxNDY3In0).

I can verify that:
- [x] User can click the "Download Payment Instructions" button at the bottom of the "Download Payment Instructions" page  to generate and download the PDF. (Similarly as within the manage obligation workflow)
- [x] On the "Pay Penalty and Track Payment(s)" page, user can see the "Outstanding Amount".
- [x] If the "Outstanding Amount" hasn't been reduced to 0, display the alert message above as in the Figma design. [(wireframe)](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?page-id=14307%3A1663&node-id=27318-245&p=f&viewport=392%2C-559%2C0.02&t=EdJyTcH0sKThe72J-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=20107%3A1497&show-proto-sidebar=1)
- [x] If the "Outstanding Amount" has been reduced to 0, display the success message above as in the Figma design. [(wireframe)](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?page-id=14307%3A1663&node-id=23398-643&p=f&viewport=392%2C-559%2C0.02&t=EdJyTcH0sKThe72J-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=20107%3A1497&show-proto-sidebar=1)

Moving to Pending PO for review of test. Thank you!

**Comment by vesselak** [comment]
Reviewed testing documentation and accepting ticket
